packages/components/src/toggle-multiple-group-control/toggle-multiple-group-control-option-icon/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import type { ForwardedRef } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ToggleGroupControlOptionIcon } from '../../toggle-group-control';
+import type { ToggleMultipleGroupControlOptionIconProps } from '../types';
+import type { WordPressComponentProps } from '../../ui/context';
+
+function UnforwardedToggleMultipleGroupControlOptionIcon(
+	{
+		isPressed,
+		...otherProps
+	}: WordPressComponentProps<
```

Comment:
Should we explicitly `Omit` the `aria-pressed` attribute from `WordPressComponentProps` here?